tsfresh/examples/test_tsfresh_baseline_dataset.py

```diff
@@ -5,8 +5,7 @@
 names and their calculated values are consistent with the known baseline.
 """
 
-from __future__ import absolute_import
-from urllib.request import urlopen
+import requests
```

Comment:
@nils-braun Totally agreed and it gives tsfresh a much simpler new data source method.  urllib is fairly primitive (apparently).  I did not want to bulk up your test-requirements.txt :)